test/testdrive/catalog.td

```diff
@@ -535,6 +532,7 @@ mz_scheduling_histogram_internal                log
 mz_scheduling_histogram_internal_1              log
 mz_scheduling_parks_internal                    log
 mz_scheduling_parks_internal_1                  log
+mz_storage_shards                               storage collection
```

Comment:
I know this predates you, but we just _have_ to find a better name for this than "storage collection." In user facing stuff we should call it a "table", I think, since all builtin tables will look the same to the user. Internally I propose we call it a "StorageManagedTable" or something like that.

Reply:
Oh I don't care so much about the internal name—just want to avoid it leaking into the output of `SHOW SOURCES`!

Reply:
so we just want to make sure this shows up as `table` here, right?

Reply:
From conversation in the storage all hands today, the consensus is that these should be `source`s; I agree with that take so am going to run with it.